xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -2935,8 +2945,8 @@ Reads exactly 8 bytes from the beginning of the span.
       <Docs>
         <param name="source">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads a <see cref="T:System.UIntPtr" /> from the beginning of a read-only span of bytes, as big endian.</summary>
+        <returns>If the span is too small to contain a <see cref="T:System.UIntPtr" />, return <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="false" /> if the span is too small to contain a <see cref="T:System.UIntPtr" />; otherwise, <see langword="true" />.</returns>
```